js/apps/admin-ui/src/user/UserForm.tsx

```diff
@@ -282,7 +282,7 @@ export const UserForm = ({
           >
             <KeycloakTextInput
               data-testid="lastName-input"
-              id="kc-lastname"
+              id="kc-lastName"
```

Comment:
Why was this `id` changed? That seems completely unrelated to the translation fix.

Reply:
This is to fix the problem that the lastname input input box cannot get focus correctly. You can see lines 278-289. Only the "n" in this one is in lowercase, and the others are all "N". I refer to its peer "firstName", on lines 265-277.

Reply:
Ahh yes, you are indeed correct. Nice catch!

Reply:
@potterhe could you create a separate PR for this fix? We can backport this to a patch release more easily that way, as the translation mechanism has changed since we released 22.

Reply:
Perfect thanks!